### DISCUSSION TITLE
Error in getMarkerFeatures

### DISCUSSION THREAD

**Discussion by mkojima123** [GreenleafLab/ArchR#1874] [Questions / Documentation]
Hi.
I have an error in `getMarkerFeatures` when using my data.
https://www.archrproject.com/bookdown/pairwise-testing-between-groups.html

```
Error in h(simpleError(msg, call)) :
    error in evaluating the argument 'x' in selecting a method for function 'assays': <SummarizedExperiment>[,j] index out of bounds: 0
```

Is it number of cells issue ? Please tell me about this error messages.

**Comment by rcorces** [comment]
Errors are meant to be posted to the Issues section, following the provided Issue Template and giving a reproducible example with the tutorial dataset. if you would like help, please re-post there.

**Reply by mkojima123** [reply]
Thank you for your prompt reply.
If something comes up I’ll let you know. Please regard this discussion closed.
